estate_account/__manifest__.py

```diff
@@ -0,0 +1,9 @@
+{
+    'name': 'Real Estate Accounting',
+    'category': 'Tutorials/Real Estate Accounting',
+    'version': '0.1',
+    'installable': True,
```

Comment:
Don't you want the module to be installed as soon as estate and account are installed? Could you find a way to imply that?